libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/RecyclerViewScrollPositionManager.java

```diff
@@ -0,0 +1,35 @@
+package org.wordpress.android.util.helpers;
+
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+
+public class RecyclerViewScrollPositionManager implements RecyclerViewScrollPositionSaver {
+    private static final String RV_POSITION = "rv_position";
+    private static final String RV_OFFSET = "rv_offset";
+    private int mRVPosition = 0;
+    private int mRVOffset = 0;
+
+    @Override
+    public void onSaveInstanceState(Bundle outState, RecyclerView recyclerView) {
+        outState.putInt(RV_POSITION,
+                ((LinearLayoutManager)recyclerView.getLayoutManager()).findFirstVisibleItemPosition());
```

Comment:
that's correct - I could add a `ClassCastException` but that's exactly what's going to happen anyway. Added a comment to warn about this in b81a748